src/rules/prefer-spy-on.ts

```diff
@@ -69,6 +69,17 @@ export default createRule({
 
         if (!jestFnCall) return;
 
+        // The `jest.fn()` must be in a `test`/`it` block in order for us to
+        // report it.
+        const ancestors = context.getAncestors();
+        const isInTestBlock = ancestors.some(
+          ancestor =>
+            ancestor.type === AST_NODE_TYPES.CallExpression &&
+            ['it', 'test'].includes(ancestor.callee.name),
```

Comment:
I think this should use the `utils` enum, so that it also considers other forms such as `xtest` & `xit` as test blocks.
(Can't remember the enum name off the top of my head :joy:)

Reply:
How do I match multiple enums with a string? `[...DescribeAlias, ...TestCaseName].includes(ancestor.callee.name)` throws `TypeError: _tsUtils.DescribeAlias is not iterable`.

Reply:
Actually, if you use `in` it'll match `hasOwnProperty` (which is what you should use instead of `in` in this case).

But actually you should use the guard we have: `isTestCase`, as it's a more accurate test :)